classes/04class/exercises/c04-iac01/ravisenevirathne/iam.tf

```diff
@@ -0,0 +1,57 @@
+resource "aws_iam_policy" "iam_policy" {
+  name = "iam_policy"
+  policy = data.aws_iam_policy_document.iam_policy_doc.json
+}
+
+data "aws_iam_policy_document" "iam_policy_doc"{
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:ListBucket"
+    ]
+    resources = [
+      aws_s3_bucket.S3.arn
+    ]
+  }
+
+    statement {
+    effect = "Allow"
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject"
+    ]
+    resources = [
+      "${aws_s3_bucket.S3.arn}/*"
+    ]
+  }
+
+}
+
+
+/*
+resource "aws_iam_policy" "iam_policy" {
```

Comment:
I'd recommend you to not add commented code to a main branch.
If you can remove those lines would be great.

Reply:
Unwanted comments have been removed from iam.tf.   Thank you